File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1542.lua
function split(inputstr, sep)
    if sep == nil then
            sep = "%s"
    end
    local t={}
    for str in string.gmatch(inputstr, "([^"..sep.."]+)") do
            table.insert(t, str)
    end
    return t
end

while true do
    linha = io.read()
    if linha == nil or linha == "0" then break end
    ls = split(linha)

    q = tonumber(ls[1])
    d = tonumber(ls[2])
    p = tonumber(ls[3])
    
    pg = (q*d*p)/(p-q);
    res = ""
    
    res = res .. math.floor(pg);
            
    if(math.floor(pg) == 1) then res = res .." pagina"
    else res  = res .. " paginas" end
    
    print(res)

end

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1542.rb
while linha = gets
        
    ls = linha.split(' ')
    q = (ls[0]).to_i
    if q == 0 then break end
    
    d = (ls[1]).to_i
    p1 = (ls[2]).to_i
    
    pg = (q*d*p1)/(p1-q);
    res = ""
    
    res += pg.floor.to_s;
            
    if(pg.floor == 1) then res += " pagina"
    else res += " paginas" end
    
    puts(res)

end